docs/tutorial.rst

```diff
@@ -33,56 +24,70 @@ Install Python 3: https://www.python.org/downloads/windows/
 
 In the installer, be sure to check the "Add Python to PATH" checkbox.
 
-Create Virtual Environment
---------------------------
-
-A Python Virtual Environment (virtualenv) is an isolated Python environment where you can install packages without modifying the system Python.  Using a virtualenv for cumulusci is recommended to avoid issues and conflicts with other applications using your system Python.
-
-macOS
+Linux
 ^^^^^
 
-.. code-block:: console
+Python 3 is installed by default on most major distributions. If it is not, install it via your package manager.
 
-    $ python3 -m venv cci
+Install CumulusCI
+-----------------
 
-This creates a virtualenv in a new folder named ``cci``. Whenever you want to work with commands installed in this virtualenv, you must first activate it:
+macOS/Linux
+^^^^^^^^^^^
 
-.. code-block:: console
+On macOS and Linux, the easiest way to install CumulusCI is using `homebrew <https://docs.brew.sh/>`_  and `linuxbrew <https://linuxbrew.sh/>`_, respectively:
```

Comment:
This will use a homebrew-installed Python, right? Should we remove the "Install Python 3" link for macOS above?

Reply:
Ah, good catch. I'll remove that macOS/Linux from that section, then move it under Windows.

Also, noticed that they merged Linuxbrew back in 2.0.0, so I'll remove any of those references.